go.mod

```diff
@@ -81,6 +81,8 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require github.com/ipfs/go-ipfs-ds-help v1.1.0
```

Comment:
Separated?

Reply:
dont know how this happened, only ran go mod tidy